amundsen_application/dashboard_preview/mode_preview.py

```diff
@@ -69,4 +69,8 @@ def _get_preview_image_url(self, *, uri: str) -> str:
         if web_preview_image_key not in result:
             raise FileNotFoundError('No preview image available on {}'.format(uri))
 
-        return result[web_preview_image_key]
+        image_url = result[web_preview_image_key]
```

Comment:
cc @jinhyukchang I was able to reproduce some dashboard preview failures in staging that all reported the same error `Encountered exception: 'NoneType' object has no attribute 'replace'`. 

I found that this was an error coming from our internal requests library -- not Mode -- and the reason was that the `web_preview_image` key sometimes exists but has `None` as the value.